spec.bs

```diff
@@ -1184,24 +1188,39 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
   1. Set |leadingBidInfo|'s [=leading bid info/component seller=] to |winningComponentConfig|'s
     [=auction config/seller=].
   1. Let « |topLevelSellerSignals|, unusedTopLevelReportResultBrowserSignals » be the result of
-    running [=report result=] with |leadingBidInfo| and |winningComponentConfig|.
+    running [=report result=] with |leadingBidInfo|, |topLevelDirectFromSellerSignalsForSeller| and
+    |winningComponentConfig|.
   1. Set |leadingBidInfo|'s [=leading bid info/auction config=] to |winningComponentConfig|.
   1. Set |leadingBidInfo|'s [=leading bid info/component seller=] to null.
   1. Set |leadingBidInfo|'s [=leading bid info/top level seller=] to |auctionConfig|'s
     [=auction config/seller=].
   1. Set |leadingBidInfo|'s [=leading bid info/top level seller signals=] to
     |topLevelSellerSignals|.
+  1. Let « |directFromSellerSignals|, |directFromSellerSignalsForSeller| »  be the result of running
+    [=get direct from seller signals=] with |winningComponentConfig|'s

```

Comment:
This algorithm expects a: `boolean, origin, string, map` but you pass in a `boolean, (null, string, promise, or failure), map`. This seems wrong?